pulp_docker/app/downloaders.py

```diff
@@ -93,26 +95,28 @@ def __init__(self, *args, **kwargs):
         Update the Bearer token to be used with all requests.
         """
         async with self.token_lock:
-            if self.registry_auth['bearer'] is not None and \
-               self.registry_auth['bearer'] == used_token:
+            if (
+                self.registry_auth["bearer"] is not None
+                and self.registry_auth["bearer"] == used_token
+            ):
                 return
             log.info("Updating bearer token")
-            bearer_info_string = response_auth_header[len("Bearer "):]
-            bearer_info_list = re.split(',(?=[^=,]+=)', bearer_info_string)
+            bearer_info_string = response_auth_header[len("Bearer ") :]
```

Comment:
Not my favorite, but they want to keep it this way. https://github.com/psf/black/issues/311